### ISSUE TITLE
"Failed to save Vaadin 7 version list to cache"

### ISSUE THREAD

**Issue by makoivis** [vaadin/eclipse-plugin-legacy#560]
**Reported by Artur Signell on 27 Jun 2013 18:38 UTC**

```
java.io.FileNotFoundException: C:\Users\Artur\.eclipse\org.eclipse.platform_4.3.0_1494577137_win32_win32_x86_64\configuration\com.vaadin.integration.eclipse\VERSIONS_7 (The system cannot find the path specified)
    at java.io.FileOutputStream.open(Native Method)
    at java.io.FileOutputStream.<init>(Unknown Source)
    at java.io.FileOutputStream.<init>(Unknown Source)
    at java.io.FileWriter.<init>(Unknown Source)
    at com.vaadin.integration.eclipse.util.network.MavenVersionManager.downloadAvailableVersionsList(MavenVersionManager.java:92)
    at com.vaadin.integration.eclipse.util.network.MavenVersionManager.getAvailableVersions(MavenVersionManager.java:47)
    at com.vaadin.integration.eclipse.configuration.VaadinFacetInstallDataModelProvider.getDefaultProperty(VaadinFacetInstallDataModelProvider.java:169)
    at com.vaadin.integration.eclipse.configuration.VaadinFacetInstallDataModelProvider.resetProperty(VaadinFacetInstallDataModelProvider.java:479)
    at com.vaadin.integration.eclipse.configuration.VaadinFacetInstallDataModelProvider.propertySet(VaadinFacetInstallDataModelProvider.java:259)
    at org.eclipse.wst.common.frameworks.internal.datamodel.DataModelImpl.internalSetProperty
```

Does the plugin forget to try to create the directory before the file? Seems to work the second time when a "download" folder has been created in the same folder.

Migrated-From: http://dev.vaadin.com//ticket/12140

**Comment by makoivis** [comment]
**Modified by Artur Signell on 15 Nov 2013 14:03 UTC**

**Comment by makoivis** [comment]
**Modified by Artur Signell on 12 May 2014 08:40 UTC**

**Comment by makoivis** [comment]
**Comment by Artur Signell on 17 Dec 2014 06:01 UTC**
A lot of tickets have been left hanging in the issue tracker through the years. Some of them are still relevant, some of them have been fixed a long time ago and some are no longer valid. To get a better look on what is important and still relevant, we are closing old tickets which have not been touched in a long time.

No further work will be done on this ticket unless someone indicates that it's still relevant. 

If this ticket is still relevant to you, please reopen it.

**Comment by makoivis** [comment]
**Comment by John Ahlroos on 30 Dec 2014 10:33 UTC**
This is really relevant, happens with our project all the time. 

See duplicate http://dev.vaadin.com/ticket/15347

**Comment by elmot** [comment]
Fixed in #655
